src/WebJobs.Script.WebHost/ContainerManagement/LinuxContainerActivityPublisher.cs

```diff
@@ -16,23 +16,27 @@ namespace Microsoft.Azure.WebJobs.Script.WebHost.ContainerManagement
 {
     public class LinuxContainerActivityPublisher : IHostedService, IDisposable, ILinuxContainerActivityPublisher
     {
+        public const string SpecializationCompleteEvent = "Specialization complete";
```

Comment:
nit: move this to `ScriptContants` and update the value to be more descriptive:
`public const string LinuxConsumptionSpecializationCompletedEvent = "Specialized";` ?

Reply:
Yes, I am thinking this is not needed outside this class (other than the test) so leaning towards leaving it here for now and move it to global constants if things change later?

Reply:
Ok to leave it here. I recommend changing it to `Specialized` or `SpecializationCompleted` - without the space

Reply:
Yes. Fixed in the current iteration.